### DISCUSSION TITLE
Event for clicking on the same value (already selected) in q-select

### DISCUSSION THREAD

**Discussion by mlesin** [quasarframework/quasar#13555] [General - Components / Directives / etc] [answered]
I'm trying to catch a moment when user clicks on the same value that was already selected in q-select.
Of course in terms of v-model nothing was changed, so i can't handle it on v-model level, but I need to trigger some side effects in my app.
Is that possible to do somehow?

**Comment by MilosPaunovic** [comment, marked as answer]
You can use `options` slot, render items and handle click event manually.

**Comment by pdanpdan** [comment]
Or you can use it in multiple mode and then you'll be notified (but you must update the model)
